site/docs/v0.13.0/enhancements/v2-worker-master.md

```diff
@@ -17,7 +17,7 @@
 
 ## How it works now 
 
-Sonobuoy uses a worker-master model, where a master delegates tasks to worker
```

Comment:
This one line from some old docs had gotten updated; I don't really care about the older ones. If its no different to anyone else I'll just leave it rather than trying to fix up other sections in this (or other older docs)

Reply:
Are these files even linked to from the site? I can't find any references to them.

Reply:
The enhancements aren't linked but they do actually exist if you manually write the URL.

They were really for internal discussion/documentation back then; I made the decision that I didn't really want/need them to be displayed in our online docs site though.